.github/workflows/build-release-devnet.yml

```diff
@@ -0,0 +1,32 @@
+name: Build and Release Devnet
+
+on:
+  push:
+    branches: [ "devnet" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: "recursive"
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Login to Docker Hub
+      run: docker login -u atletanetwork -p ${{ secrets.REGISTRYPASS }} docker.io
+
+
+    - name: Build and Push Docker Image
+      run: |
+          DATE=$(date +%d%m%y)
```

Comment:
This timestamp string can be built in single call like `TIMESTAMP=$(date +%d%m%y-%H%M%S)`